types/ember__routing/-private/route-info.d.ts

```diff
@@ -0,0 +1,47 @@
+// https://api.emberjs.com/ember/3.6/classes/RouteInfo
+/**
+ * A `RouteInfo` is an object that contains metadata about a specific route within a `Transition`.
+ * It is read-only and internally immutable.
+ * It is also not observable, because a `Transition` instance is never changed after creation.
+ */
+export default class RouteInfo {
```

Comment:
Is there a good reason to make this a class instead of an interface?

Reply:
I just made it a class because that's how it's defined in the Ember docs. I can definitely change that to an interface.